frontend/packages/topology/src/__tests__/TopologyShortcuts.spec.tsx

```diff
@@ -0,0 +1,136 @@
+import { shallow } from 'enzyme';
+import { Shortcut } from '@console/shared';
+import { TopologyViewType } from '../topology-types';
+import { getTopologyShortcuts } from '../components/graph-view/TopologyShortcuts';
+
+describe('TopologyShortcuts tests', () => {
+  it('should show reduced list in view shortcuts popover on topology toolbar when there are no workloads', () => {
+    const wrapper = shallow(
+      getTopologyShortcuts(jest.fn(), {
+        supportedFileTypes: undefined,
+        isEmptyModel: true,
+        viewType: TopologyViewType.graph,
+      }),
+    );
+    const secondWrapper = shallow(wrapper.getElement());
+    expect(secondWrapper.find(Shortcut).exists()).toBe(true);
+    expect(secondWrapper.find(Shortcut).props().ctrl).toBe(true);
```

Comment:
Ideally we shouldn't be looking inside props of a component rendered while testing.
but I couldn't think of a way in which we could avoid that for testing this util.
@andrewballantyne WDYT?

Reply:
Agreed, tests that dig into components is a bad idea. These tests are really getting nitty and gritty into the components and the way they are constructed just to see if they are there.

How about `data-test-id`s? `Shortcut` does not provide that functionality yet, but it would be easy to add, perhaps just a spread of any unknown props to the `tr`.

Then to test, it would be just a selection of `data-test-id`s to see if the shortcuts are there.

Reply:
Yes, this seems like a good alternative. We could use `data-test-id`s to facilitate the tests, since we just need to check the presence of the components based on the conditions.